src/components/DataDetails.vue

```diff
@@ -30,7 +30,9 @@ v-card.data-details
       title="Actions",
       :clickable="true",
       :rows="actions",
-      :color="$vuetify.theme.themes.light.secondary.lighten5",
+      :color="$vuetify.theme.dark?\
+        $vuetify.theme.themes.dark.secondary.darken3:\
+        $vuetify.theme.themes.light.secondary.lighten5",
```

Comment:
We should make a semantic color for this using a custom key in each theme, see what I've done in 7bc217a9f4f162489b1cf2a7e6da358c76a88274

Reply:
Since we will have only dark and light two themes, would that be an overkill, and it may not work well in IE?

Reply:
The theme shouldn't be referenced explicitly like this. Rather, it should just be `color="key"`, where key is some value that is present in each theme.

Reply:
I thought using the theme color from the will make theming more easy to change. 
For example, If the user says that the secondary color is green, it will be a shade of green. Otherwise, the user needs to explicitly overwrite that key?

Reply:
The user could use the theme generator and only pick the predefined colors. https://theme-generator.vuetifyjs.com/

Reply:
@subdavis what do you think? I guess dark vs. light may need to be treated independently from dynamic dispatch of theme colors, but generally I would prefer to avoid these kinds of switch statements in components.

Reply:
`$vuetify.theme.currentTheme.secondary` seems to be what you want here.

Reply:
@subdavis semantic colors? Could you explain a little bit more?

Reply:
right now we have primary, secondary, accent, etc.

But we do variations on those based on whether the theme is light or dark:

```js
$vuetify.theme.dark?\
        $vuetify.theme.themes.dark.secondary.darken3:\
        $vuetify.theme.themes.light.secondary.lighten5
```

@zachmullen has suggested and I think I agree that this should be avoided.

In most cases, we should be using `$vuetify.theme.currentTheme.<identifier>` so that we never have to add logic regarding light vs dark.  This solution doesn't help with the example above, though.

I think the best thing to do would be to add another named color in `vuetifyConfig.js` called `highlightedListItem` or something like that and specify an explicit value for it in each theme rather than trying to derive it from another color.

Another alternative is to drop the custom background styling on `DetailList.vue`.  It doesn't look as good (IMO) but fighting Vuetify never seems to result in clean code.

For now, I vote we drop the `color` prop and custom styling in `DetailList.vue`

Reply:
@subdavis Cool, thank you for the clarification. 
We currently have another instance that needs this kind of theme checking. 
What do think about this case?
https://github.com/girder/girder_web_components/pull/197/files#diff-20a85c2a028ea24e3d92a12b008ceda7R302